js/modal.js

```diff
@@ -76,6 +79,7 @@
           var oldWidth = $body.innerWidth();
           $body.css('overflow', 'hidden');
           $body.width(oldWidth);
+          $('html').css('overflow', 'hidden');
```

Comment:
I guess we can add `$('html')` to https://github.com/Greta/materialize/blob/71fe188a14b091c1a415e35cb9a34480b905f58c/js/modal.js#L78 to cache the lookup.

Then you can merge this with line 80.